features/resources/spatial_DecompositionTable.csv

```diff
@@ -0,0 +1,46 @@
+ApplicableEntity,RelatedObjects
+IfcBridgePart,IfcSpace
+IfcBridge,IfcBridge
+IfcBridge,IfcBridgePart
+IfcBridge,IfcSpace
+IfcBuilding,IfcBuilding
+IfcBuilding,IfcBuildingStorey
+IfcBuilding,IfcSpace
+IfcBuildingStorey,IfcBuildingStorey
+IfcBuildingStorey,IfcSpace
+IfcExternalSpatialElement,IfcExternalSpatialElement
+IfcFacilityPartCommon,IfcSpace
+IfcFacility,IfcFacility
+IfcFacility,IfcFacilityPartCommon
+IfcFacility,IfcSpace
+IfcMarineFacility,IfcMarineFacility
+IfcMarineFacility,IfcMarinePart
+IfcMarineFacility,IfcSpace
+IfcMarinePart,IfcSpace
+IfcProject,IfcSite
+IfcProject,IfcBuilding
+IfcProject,IfcFacility
+IfcProject,IfcBridge
+IfcProject,IfcMarineFacility
+IfcProject,IfcRailway
+IfcProject,IfcRoad
+IfcProject,IfcExternalSpatialElement
+IfcProject,IfcSpace
```

Comment:
`IfcProject` **is not** an `IfcSpatialElement`. This contradicts the rule's documentation.

Reply:
Good point. Changing the documentation to:
```

  Scenario: Agreement on each IfcSpatialElement being aggregated as per spatial composition table.

      Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
      And An IfcSpatialElement
      Then Each IfcSpatialElement must be aggregated as per spatial_CompositionTable.csv
```